manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/pipeline.yaml

```diff
@@ -624,11 +624,7 @@ spec:
             # Following environment variables are only needed when using Cloud SQL and GCS.
             {{ if .Values.managedstorage.enabled }}
             - name: OBJECTSTORECONFIG_BUCKETNAME
-              {{ if .Values.managedstorage.databaseNamePrefix }}
```

Comment:
I think you already aware, before this change, we don't allow user to specify the gcs bucket name. 
The gcs bucket name is auto inferred from the cloud sql instance name. The reason for that is so the cloudsql and GCS data can alway be in sync. When user reinstall, the data can be recovered correctly. 

Allowing user to specify gcs bucket name might break that assumption and causing unexpected behavior.

Reply:
Thanks. A fast fix is to copy the bucket-gen code here to argo configmap.(although it would lead to code dup. anyway to dedup the code would be sweet)

Reply:
What do you think about my approach to avoid code dupe?

Reply:
I'm OK if it works